### DISCUSSION TITLE
无法扫码登陆

### DISCUSSION THREAD

**Discussion by gyhmz** [ywmoyue/biliuwp-lite#325] [General]
有二维码但是扫不出来

**Comment by ywmoyue** [comment]
扫不出来是什么意思，具体表现是什么

**Reply by gyhmz** [reply]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/95202722/1ecc1422-f34f-4089-a673-d37b4c3302f0)

字面意思，扫不出来，版本是最新版arm64

**Reply by ywmoyue** [reply]
是否有x86设备可以试试x86能否扫码登录呢，我这边没有arm的windows设备

**Comment by gyhmz** [comment]
X86 32位和64位都正常
![image](https://github.com/ywmoyue/biliuwp-lite/assets/95202722/50bf2dea-2e90-4c6a-ab83-84668cedd54c)

**Comment by ywmoyue** [comment]
@gyhmz Arm的密码登录和短信登录是否可用

**Reply by gyhmz** [reply]
可用

**Comment by ywmoyue** [comment]
@gyhmz 麻烦试一下4.6.23版本的二维码扫码是否可用

**Reply by luoquan-30** [reply]
我这边使用4.6.23是无法使用的，密码登录时需要验证码，但是验证码无法发送。目前可用登录方式仅有验证码登录

**Reply by ywmoyue** [reply]
@luoquan-30 ARM架构的话再试试这个https://github.com/ywmoyue/TempFileResource/releases/tag/debug_biliuwp_4.6.23.1

**Reply by luoquan-30** [reply]
这个版本是可以使用的，非常感谢
